index.d.ts

```diff
@@ -0,0 +1,36 @@
+declare namespace execall {
+	interface Match {
+		match: string;
+		sub: string[];
+		index: number;
+	}
+}
+
+/**
+Find multiple RegExp matches in a string.
+
+@param regexp - Regular expression to match against the `string`.
+@returns An array of matches.
+
+@example
+```
+import execall = require('execall');
+
+execall(/(\d+)/g, '$200 and $400');
+// [
+// 	{
+// 		match: '200',
+// 		sub: ['200'],
```

Comment:
What do you think about renaming this to `subMatch`? I feel that `sub` is not clear enough. I was very sloppy with naming some years back...